moped-editor/src/views/projects/projectView/ProjectComponents/index.js

```diff
@@ -150,8 +176,8 @@ export default function MapView({ projectName, phaseKey, phaseName }) {
 
   /* fits clickedComponent to map bounds - called from component list item secondary action */
   const onClickZoomToComponent = (component) => {
-    const componentId = component.project_component_id;
-    const featureCollection = featureCollectionsByComponentId[componentId];
+    const features = getAllComponentFeatures(component);
+    const featureCollection = { type: "FeatureCollection", features };
```

Comment:
This used to get a feature collection from `featureCollectionsByComponentId` which is a map of feature collections  by project component ID. It was easier to adapt this to build a feature collection from a component features which are already in memory for project components and the new related components.